Scala/Scala.sublime-syntax

```diff
@@ -147,6 +147,46 @@ contexts:
     - match: '{{typeprefix}}'
       push: single-type-expression
 
+  annotation:
+    - match: '(@){{plainid}}\.'
+      captures:
+        0: storage.modifier.annotation.scala
```

Comment:
Stylistically we've been moving away from `0` captures and instead using `scope: ` for when a scope applies to the whole match.